packages/amplify-graphql-schema-generator/src/utils/__tests__/vpc-helper.test.ts

```diff
@@ -0,0 +1,248 @@
+import {
+  RDSClient,
+  DescribeDBClustersCommandOutput,
+  DescribeDBInstancesCommandOutput,
+  DescribeDBSubnetGroupsCommandOutput,
+} from '@aws-sdk/client-rds';
+
+import {
+  getHostVpc,
+} from '../vpc-helper';
+
+
+const sendSpy = jest.spyOn(RDSClient.prototype, 'send');
+
+describe('detect VPC settings', () => {
+
+  it('should detect VPC settings for an RDS instance', async () => {
+    // TS complains about resolving values in the spy.
+    // Cast it through never to overcome the compile error.
+    sendSpy
+      .mockResolvedValueOnce(instanceResponse as never);
+
+    const result = await getHostVpc(
+      'mock-rds-cluster-instance-1.aaaaaaaaaaaa.us-west-2.rds.amazonaws.com',
+      'us-west-2'
+    );
+
+    expect(result).toBeDefined();
```

Comment:
is there a reason to not have a stronger assertion about the output here that it should match the VPC config that's mocked in the response?

Reply:
:sigh: I forgot to go back and assert the actual data. :(  Thanks for catching, will update.